samples/persistent-subscriptions/Program.cs

```diff
@@ -115,6 +123,108 @@ class Program
+			    "subscription-group",
+			    userCredentials);
+		    
+		    Console.WriteLine($"GroupName: {info.GroupName}, EventSource: {info.EventSource}, Status: {info.Status}");
+
+		    #endregion get-persistent-subscription-to-stream-info
+	    }
+	    
+	    static async Task GetPersistentSubscriptionToAllInfo(EventStorePersistentSubscriptionsClient client) {
+		    #region get-persistent-subscription-to-all-info
+
+		    var userCredentials = new UserCredentials("admin", "changeit");
+		    var info = await client.GetInfoToAllAsync(
+			    "subscription-group",
+			    userCredentials);
+		    
+		    Console.WriteLine($"GroupName: {info.GroupName}, EventSource: {info.EventSource}, Status: {info.Status}");
+
+		    #endregion get-persistent-subscription-to-all-info
+	    }
+	    
+	    static async Task ReplayParkedToStream(EventStorePersistentSubscriptionsClient client) {
+		    #region replay-parked-of-persistent-subscription-to-stream
+
+		    var userCredentials = new UserCredentials("admin", "changeit");
+		    await client.ReplayParkedToStreamAsync(
+			    "test-stream",
+			    "subscription-group",
+			    numberOfEvents: 10,
+			    userCredentials);
+		    
+		    #endregion persistent-subscription-replay-parked-to-stream
+	    }
+	    
+	    static async Task ReplayParkedToAll(EventStorePersistentSubscriptionsClient client) {
+		    #region replay-parked-of-persistent-subscription-to-all
+
+		    var userCredentials = new UserCredentials("admin", "changeit");
+		    await client.ReplayParkedToAllAsync(
+			    "subscription-group",
+			    numberOfEvents: 10,
+			    userCredentials);
+		    
+		    #endregion replay-parked-of-persistent-subscription-to-all
+	    }
+	    
+	    static async Task ListPersistentSubscriptionsToStream(EventStorePersistentSubscriptionsClient client) {
+		    #region list-persistent-subscriptions-to-stream
+
+		    var userCredentials = new UserCredentials("admin", "changeit");
+		    var subscriptions = await client.ListToStreamAsync(
+			    "test-stream",
+			    userCredentials);
+
+		    foreach (var s in subscriptions) {
+			    Console.WriteLine($"GroupName: {s.GroupName}, EventSource: {s.EventSource}, Status: {s.Status}");
+		    }
+		    
+		    #endregion list-persistent-subscriptions-to-stream
+	    }
+	    
+	    static async Task ListPersistentSubscriptionsToAll(EventStorePersistentSubscriptionsClient client) {
+		    #region list-persistent-subscriptions-to-all
+
+		    var userCredentials = new UserCredentials("admin", "changeit");
+		    var subscriptions = await client.ListToAllAsync(userCredentials);
+
+		    foreach (var s in subscriptions) {
+			    Console.WriteLine($"GroupName: {s.GroupName}, EventSource: {s.EventSource}, Status: {s.Status}");
+		    }
+		    
+		    #endregion list-persistent-subscriptions-to-all
+	    }
+	    
+	    static async Task ListAllPersistentSubscriptions(EventStorePersistentSubscriptionsClient client) {
+		    #region list-all-persistent-subscriptions
```

Comment:
**FIX:** As I see other clients are using `list-persistent-subscriptions` for this snippet. See, e.g.:
- https://github.com/EventStore/EventStoreDB-Client-Rust/blob/master/examples/persistent_subscriptions.rs#L190
- https://github.com/EventStore/EventStore-Client-NodeJS/pull/273/files#diff-1ac0f13c516056c662743ace8bed2addc43bbcb95d09da0ca05ff983b37c58b9R370